tests/test_notifications.py

```diff
@@ -681,7 +685,7 @@ def test_serialize_node_level_event_that_adopts_parent_settings(self):
         self.project_subscription.save()
         self.node.add_contributor(contributor=user, permissions=['read'])
         self.node.save()
-        node_subscriptions = [x for x in utils.get_all_node_subscriptions(user, self.node)]
+        node_subscriptions = [x for x in utils.get_all_node_subscriptions(user, self.node) if x.event_name != 'mailing_list_events']
```

Comment:
`serialize_event` takes a single subscription, not a list of them. I have no idea why an empty list was being passed in originally, but I've removed it.